### DISCUSSION TITLE
Show "NOTES" from days meal

### DISCUSSION THREAD

**Discussion by niklaspoulsenSH** [firstof9/lovelace-grocy-meal-plan-card#11] [Ideas]
Hi, Would it be possible to update code, so it also shows notes made in the GROCY meal plan? :)

**Comment by firstof9** [comment]
Should be doable.

**Comment by firstof9** [comment]
Looks like notes are already supported.

**Reply by niklaspoulsenSH** [reply]
When i add a note its empty - and is added as a box - not in the box with days meal

**Reply by firstof9** [reply]
I see, I'll take a look.
